crates/red_knot_python_semantic/resources/mdtest/import/errors.md

```diff
@@ -55,3 +55,27 @@ from b import x
 
 x = "foo"  # error: [invalid-assignment] "Object of type `Literal["foo"]"
 ```
+
+## Invalid syntax
```

Comment:
As @AlexWaygood mentioned, we'll probably need to put this in its own file with `invalid_syntax` in the name, e.g. `import/invalid_syntax.md`